src/timely-util/src/containers.rs

```diff
@@ -217,6 +231,28 @@ mod builder {
         pending: VecDeque<Column<C>>,
     }
 
+    /// Returns `true` if columnar allocations should come from lgalloc.
+    #[inline]
+    pub fn enable_lgalloc_columnar() -> bool {
+        ENABLE_LGALLOC_COLUMNAR.with(|enabled| *enabled.borrow())
+    }
+
+    /// Set whether columnar allocations should come from lgalloc. Applies to future allocations.
+    pub fn set_enable_lgalloc_columnar(enabled: bool) {
+        ENABLE_LGALLOC_COLUMNAR.with(|enable| *enable.borrow_mut() = enabled);
+    }
+
+    /// Allocate a region of memory with a capacity of at least `len` that is aligned to 8 bytes
+    /// and zeroed.
+    #[inline]
+    pub(crate) fn alloc_aligned_zeroed(len: usize) -> Region<u64> {
+        if enable_lgalloc_columnar() {
+            Region::new_auto_zeroed(len)
+        } else {
+            Region::new_heap_zeroed(len)
+        }
+    }
```

Comment:
Mind adding to this comment that it's aligned to 8 bytes because it's a Region of `u64`s?